bin/index.js

```diff
@@ -49,7 +51,7 @@ try {
   main = require(path.join(__dirname, '..', 'commands', `${command}.js`))
 } catch (err) {
   console.error(chalk.red(`
-Unknown command: ${command}`))
+Unknown command: ${command}`, err))
```

Comment:
Yeah agreed, my only worry is the buildbot cli currently already has a `--debug` flag for the `bm buildbot build` cmd. So prob wont be able to make this a consistent pattern without changing the buildbot cmd